### PR TITLE
/sectors command and Ace permissions

### PR DESCRIPTION
**Pull request by Helpme0822**
Removes the keybind and adds a /sectors command. Also adds ace permissions for the /sectors command.

### CODE REVIEW

File: server.lua [hunk 1]
```diff
@@ -1,3 +1,12 @@
+RegisterServerEvent("sectors.getIsAllowed")
+AddEventHandler("sectors.getIsAllowed", function()
+    if IsPlayerAceAllowed(source, "command.sectors") then
```
Comment (DyrekKing) [inline server.lua:3]: You can just do.
```lua
local isAllowed = IsPlayerAceAllowed(source, "command.sectors")
TriggerClientEvent("sectors.returnIsAllowed", source, isAllowed )

**Review by adamragasits — CHANGES_REQUESTED** [review]
```lua
local isAllowed = IsPlayerAceAllowed(source, "command.sectors")
TriggerClientEvent("sectors.returnIsAllowed", source, isAllowed )
```
This is a better and easier approach.
server.lua:3-7

**Comment by DyrekKing** [comment]
If you could change server.lua as I suggested, I'll merge it.

**Review by adamragasits — CHANGES_REQUESTED** [review]
Remove this thread completely, it has no use.
client.lua:29-37

**Comment by Helpme0822** [comment]
Requested changes have been done

**Comment by adamragasits** [comment]
Awesome! Thank you!

**Pull request merged**